pkgs/games/wolfstoneextract/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, stdenv
+, fetchFromBitbucket
+, cmake
+}:
+stdenv.mkDerivation rec {
+  pname = "wolfstoneextract";
+  version = "1.2";
+
+  src = fetchFromBitbucket {
+    owner = "ecwolf";
+    repo = "wolfstoneextract";
+    rev = version;
+    hash = "sha256-yrYLP2ewOtiry+EgH1IEaxz2Q55mqQ6mRGSxzVUnJ8Q=";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  meta = with lib; {
+    description = "Utility to extract Wolfstone data from Wolfenstein II";
+    homepage = "https://bitbucket.org/ecwolf/wolfstoneextract/src/master/";
+    license = licenses.gpl3Only;
```

```suggestion
    license = with licenses; [ gpl3Only bsd3 ];
```
It seems to include bsd3.
https://bitbucket.org/ecwolf/wolfstoneextract/src/e1d37f2ad5b5207431fa6328cc0f6602960ebf66/src/doomerrors.h#lines-5:31